interface/interface.dm

```diff
@@ -27,6 +27,21 @@
 		to_chat(src, "<span class='danger'>The forum URL is not set in the server configuration.</span>")
 	return
 
+/client/verb/demoview(roundnumber as num)
+	set name = "demoview"
+	set desc = "Enter Round Number you wish to see the replay of"
+	set hidden = 1
+	var/roundurl = CONFIG_GET(string/demourl)
+	if(roundurl)
+		if(roundnumber)
+			if(alert("This will open the demo in your browser. Are you sure?",,"Yes","No")!="Yes")
```

Comment:
Move above the if(roundnumber) 
Currently it only asks if you've entered a round ID, if you haven't you just get sent without confirmation

Maybe add an isnum check for both GLOB.round_id - 1 and var/roundnumber incase DB is down or other weird fuckery